specs/deneb/polynomial-commitments.md

```diff
@@ -83,6 +83,7 @@ Public functions MUST accept raw bytes as input and perform the required cryptog
 | - | - |
 | `FIELD_ELEMENTS_PER_BLOB` | `uint64(4096)` |
 | `FIAT_SHAMIR_PROTOCOL_DOMAIN` | `b'FSBLOBVERIFY_V1_'` |
+| `RANDOM_CHALLENGE_KZG_MULTI_DOMAIN` | `b'RCKZGMULTI___V1_'` |
```

Comment:
Making this look like the one above

```suggestion
| `RANDOM_CHALLENGE_KZG_MULTI_DOMAIN` | `b'RCKZGMULTI_V1_'` |
```

Reply:
I think it's that way so the value is 16 bytes.

Reply:
Yeah :)

Reply:
ACK. Let's leave it as is then.